src/Compilers/Core/Portable/Symbols/IFieldSymbol.cs

```diff
@@ -48,6 +48,14 @@ public interface IFieldSymbol : ISymbol
         /// </summary>
         bool IsFixedSizeBuffer { get; }
 
+        /// <summary>
+        /// If IsFixedSizeBuffer is true, the value between brackets in the fixed-size-buffer declaration.
+        /// If IsFixedSizeBuffer is false FixedSize is 0.
```

Comment:
Should we also specify that FixedSize is 0 in error scenarios? i.e. if a bad constant value is used in source.